scripts/ios/build.sh

```diff
@@ -45,8 +45,15 @@ for TARGET_ARCHITECTURE in arm64; do
   export UNAME=Darwin
   export LAPACK=''
 
-  export OPENCV_DIR=$BUILD_DIR/opencv/build/build-$TARGET_ARCHITECTURE-iphoneos/install
+  export OPENCV_DIR=$BUILD_DIR/lib/opencv/build/build-$TARGET_ARCHITECTURE-iphoneos/install
 
   export USE_SLAM
+
+  # If we enable clear for ./scripts/build.sh, it will nuke OpenCV build we just did
+  export DO_CLEAR=OFF
```

Comment:
Can this be fixed in some other way? For example by doing the iOS OpenCV build in `scripts/components/opencv.sh`? Now the "command line parameter" `DO_CLEAR` defined at top of this file doesn't work like in the host and android build scripts.